train.py

```diff
@@ -1942,7 +1942,14 @@ def main():
                             width=latents.shape[3],
                         )
                         guidance_scale = 3  # >>> ????? <<<
-                        if transformer.config.guidance_embeds:
+                        original_config = transformer.config if hasattr(transformer, 'config') else None
```

Comment:
i would just set this to `None`

```suggestion
                        transformer_config = None
```